tools/get-images.sh

```diff
@@ -4,6 +4,6 @@
 #
 # dynamic list
 IMAGE_LIST=()
-IMAGE_LIST+=($(find -type f -name metadata.yaml -exec yq '.resources | to_entries | .[] | .value | ."upstream-source"' {} \;))
+IMAGE_LIST+=($(find -type f -name config.yaml -exec yq '.options.*-image |  select(.) | .default' {} \;))
```

Comment:
This is failing for me.
```bash
./tools/get-images.sh
jq: error: syntax error, unexpected '*', expecting FORMAT or QQSTRING_START or '[' (Unix shell quoting issues?) at <top-level>, line 1:
.options.*-image |  select(.) | .default         
jq: 1 compile error
```

Reply:
I have created an issue for it [here](https://github.com/canonical/kfp-operators/issues/562#issue-2492407938), basically this is related to the version of `yq` used (I'm assuming you installed yq with `pip`). The solution is to use the `snap` version of `yq`.

Reply:
You are right, I had also yq installed by pip and snap. Removing version 3.1.0 installed by pip, and it's working fine :+1: